Formula/man-db.rb

```diff
@@ -0,0 +1,78 @@
+class ManDb < Formula
+  desc "Unix documentation system"
+  homepage "http://man-db.nongnu.org/"
+  url "https://download.savannah.gnu.org/releases/man-db/man-db-2.8.5.tar.xz"
+  sha256 "b64d52747534f1fe873b2876eb7f01319985309d5d7da319d2bc52ba1e73f6c1"
+
+  keg_only :provided_by_macos
+
+  depends_on "pkg-config" => :build
+
+  resource "libpipeline" do
+    url "https://download.savannah.gnu.org/releases/libpipeline/libpipeline-1.5.1.tar.gz"
+    sha256 "d633706b7d845f08b42bc66ddbe845d57e726bf89298e2cee29f09577e2f902f"
+  end
+
+  def install
+    resource("libpipeline").stage do
+      system "./configure",
+        "--disable-dependency-tracking",
+        "--disable-silent-rules",
+        "--prefix=#{prefix}"
+      system "make"
+      system "make", "install"
+    end
+
+    ENV["libpipeline_CFLAGS"] = "-I#{include}"
+    ENV["libpipeline_LIBS"] = "-L#{lib} -lpipeline"
+
+    args = %W[
+      --disable-dependency-tracking
+      --disable-silent-rules
+      --prefix=#{prefix}
+      --disable-cache-owner
+      --disable-setuid
+      --enable-static
+      --disable-shared
```

Comment:
These two lines need to be moved to the `./configure` of `libpipeline` rather than `man-db`.